### DISCUSSION TITLE
HasOne relation in table

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#1941] [Help] [answered]
Hello,

I would like to create a resource table in that way that I can also show a value of a property of a `HasOne` relationship. How can I achieve this? I didn't find anything in the docs.

**Comment by ryangjchandler** [comment, marked as answer]
@Wulfheart You can use dot notation - `TextColumn::make('relation.column')`

**Reply by Wulfheart** [reply]
Thanks. That was also my first guess but it didn't work due to a typo.

**Comment by CreateWithCoding** [comment]
How to do HasOne (One-to-One) within the Forms ?

**Reply by francisakortia** [reply]
> How to do HasOne (One-to-One) within the Forms ?

Can anyone offer help on this?

**Reply by francisakortia** [reply]
Figured it out.

**Reply by gtodica** [reply]
> 

Hi [francisakortia](https://github.com/francisakortia), could you share your experience HasOne(One-to-One) within the Forms? I have the same issue but didn't find any information how to do it.
Thank you in advance.

**Reply by danharrin** [reply]
#2304

**Comment by putrafajarh** [comment]
Person Detail,


```
Fieldset::make('detail')
                    ->label('Details')
                    ->relationship('detail')
                    ->schema([
                        TextInput::make('instagram')
                            ->label('Instagram')
                            ->url(),
                        DatePicker::make('dob')
                            ->label('Date of Birth'),
                    ])
                    ->visibleOn('edit')
```

**Reply by Wulfheart** [reply]
What is the concrete question?
